libs/dh/market-participant/actors/feature-delegation/src/lib/table/dh-delegation-table.componen.ts

```diff
@@ -66,7 +66,11 @@ import { DhDelegationStopModalComponent } from '../stop/dh-delegation-stop-modal
           @if (entry.status === 'CANCELLED') {
             {{ null | dhEmDashFallback }}
           } @else {
-            {{ entry.validPeriod | wattDate: 'short' }}
+            {{ entry.validPeriod.start | wattDate: 'short' }}
+            @if (entry.validPeriod.end) {
+              {{ entry.validPeriod.end | wattDate: 'short' }}
+              -
+            }
```

Comment:
suggestion: let's move the dash before the date :)
```suggestion
            @if (entry.validPeriod.end) {
              -
              {{ entry.validPeriod.end | wattDate: 'short' }}
            }
```